plugins/inputs/redis_sentinel/README.md

```diff
@@ -0,0 +1,208 @@
+# Telegraf Plugin: Redis Sentinel
+
+A plugin for Redis Sentinel to monitor multiple Sentinel instances that are
+mointoring multiple Redis servers and replicas.
+
+### Configuration:
+
+```
+# Read Redis Sentinel's basic status information
+[[inputs.redis_sentinel]]
+  ## specify servers via a url matching:
+  ##  [protocol://][:password]@address[:port]
+  ##  e.g.
+  ##    tcp://localhost:26379
+  ##    tcp://:password@192.168.99.100
+  ##
+  ## If no servers are specified, then localhost is used as the host.
+  ## If no port is specified, 26379 is used
+  servers = ["tcp://localhost:26379"]
+
+  ## specify server password
+  # password = "s#cr@t%"
+
+  ## Optional TLS Config
+  # tls_ca = "/etc/telegraf/ca.pem"
+  # tls_cert = "/etc/telegraf/cert.pem"
+  # tls_key = "/etc/telegraf/key.pem"
+  ## Use TLS but skip chain & host verification
+  # insecure_skip_verify = true
+```
+
+### Measurements & Fields:
+
+The plugin gathers the results of these commands and measurements:
+
+* `sentinel masters` - `redis_sentinel_masters`
+* `sentinel sentinels` - `redis_sentinels`
+* `sentinel replicas` - `redis_replicas`
+* `info all` - `redis_sentinel`
+
+The `has-quorum` (int) field in `redis_sentinel_masters` is from calling the command `sentinels ckquorum`. `1` is for true, `0` is for false.
```

Comment:
Why not converting those values to `true` and `false` internally?

Reply:
@srebhan Addressed in https://github.com/influxdata/telegraf/pull/10042/commits/bfacfe249c2252d9c00f471a0adde08cbaba8b34